ide-migration/server/migration/api/migration-service.js

```diff
@@ -271,9 +273,9 @@ class MigrationService {
         const project = workspace.getProject(projectName)
         const projectFile = project.createFile(relativePath);
         const resource = repositoryManager.getResource(repositoryPath);
-
+        const xskModificator = new XSKProjectMigrationInterceptor();
         if (relativePath.endsWith('hdbcalculationview') || relativePath.endsWith('calculationview') || repositoryPath.endsWith('hdbcalculationview') || repositoryPath.endsWith('calculationview')) {
-            projectFile.setContent(calculationViewModifier.removeTypeArtifact(resource.getContent()));
+            projectFile.setContent(xskModificator.modify(resource.getContent()));
```

Comment:
Please, extract `xskModificator.modify(resource.getContent())` into a variable and use it in `setContent`